### PR TITLE
Arvinth/align timestamp

### PR DESCRIPTION
**Pull request by ArvinthC3000**
### Description: 

In Automate HA, the user who creates a bundle (including frontend and backend) wants to see the latest version timestamp in the bundle name. With the current change, when they create the bundle using the non-airgap method, the name will contain the timestamp (or build version) of the latest build instead of the downloaded time's timestamp.

Note: 
- Latest build version is obtained by the 'chef-automate info' command is used.
- Applicable but both internet and non-internet environments.

Observations:
- On running deploy/upgrade commands, the name of the bundles inside transfer_files are appended with the latest release version (timestamp)

Method | Result
-- | --
Airgap - Deploy | Working as expected
Airgap - Upgrade | Working as expected
Non-airgap - Deploy | Working as expected
Non-airgap - Upgrade | Working as expected

<br data-cke-eol="1">

### :chains: Related Resources

Link to Jira Ticket: https://chefio.atlassian.net/browse/MADROX-148

### :+1: Definition of Done
It's dev done and tested

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: terraform/a2ha-terraform/Makefile [hunk 1]
```diff
@@ -6,10 +6,10 @@ SUP_KEY_SH = $(shell pwd)/../../scripts/sup-keys.sh
 TMPOUT = '/tmp/make.out'
 REPO_ROOT = $(shell pwd)
 DATESTAMP := $(shell date +"%Y%m%d%H%M%S")
-BUNDLEAIB = $(TERRAFORM_PATH)/transfer_files/bundle-$(DATESTAMP).aib
-BACKENDAIB = $(TERRAFORM_PATH)/transfer_files/backend-$(DATESTAMP).aib
+BUNDLEAIB = $(TERRAFORM_PATH)/transfer_files/bundle.aib
```
Comment (FaizanSRE) [inline terraform/a2ha-terraform/Makefile:9]: We can remove this input decleration here BUNDLEAIB,FRONTENDAIB, BACKENDAIB

File: terraform/a2ha-terraform/Makefile [hunk 2]
```diff
@@ -114,19 +114,15 @@ sort:
 
 $(BUNDLEAIB):
 	@echo "# Updating FE and BE component package versions" && \
-		$(AIRGAP_SH) -t all -o $(FRONTENDAIB) -b $(BACKENDAIB) -v $(FRONTENDAIB_TFVARS) -q $(BACKENDAIB_TFVARS)  && \
-	  $$(hab pkg path core/coreutils)/bin/md5sum $(FRONTENDAIB) > $(FRONTENDAIB).md5 && \
-	  $$(hab pkg path core/coreutils)/bin/md5sum $(BACKENDAIB) > $(BACKENDAIB).md5
+		$(AIRGAP_SH) -t all -o $(FRONTENDAIB) -b $(BACKENDAIB) -v $(FRONTENDAIB_TFVARS) -q $(BACKENDAIB_TFVARS)
```
Comment (FaizanSRE) [inline terraform/a2ha-terraform/Makefile:117]: here at -o option pass terraform path instead

File: scripts/bundle_creation.sh [hunk 3]
```diff
@@ -85,27 +88,32 @@ airgap_bundle_create() {
   args+=("${original_aib_path}")
   # printf '%s\n' "Running: ${CHEF_AUTOMATE_BIN_PATH} ${args[*]}"
   if "${CHEF_AUTOMATE_BIN_PATH}" "${args[@]}" > /tmp/thelog.log; then
+    BUILD_VERSION=`"${CHEF_AUTOMATE_BIN_PATH}" airgap bundle info "${original_aib_path}" | grep -i version | awk '{print $2}'`
     if [ "$BUNDLE_TYPE" == "upgradefrontends" ] || [ "$BUNDLE_TYPE" == "all" ]
     then
-          cat "${original_aib_path}" > "${TARBALL_PATH}"
-          outfile=${ORIGINAL_TARBALL:-${TARBALL_PATH}}
+          frontend_aib="/hab/a2_deploy_workspace/terraform/transfer_files/frontend-${BUILD_VERSION}.aib"
```
Comment (FaizanSRE) [inline scripts/bundle_creation.sh:94]: frontend_aib="${terraform path what we will get from makefile} transfer_files/frontend-${BUILD_VERSION}.aib"

File: terraform/a2ha-terraform/Makefile (hunk 2 above)
Comment (ArvinthC3000) [reply terraform/a2ha-terraform/Makefile:117]: Done

**Review by FaizanSRE — APPROVED** [review]

**Pull request closed without merging**